pages/datasets/[datasetId].vue

```diff
@@ -764,7 +763,31 @@ export default {
             throw error
           })
       }
-    }
+    },
+    getOsparcLink: async function () {
+      try {
+        if(this.$config.public.osparc_disable_hornet_button) {
```

Comment:
osparc_enable_check_hornet_manifest needs to be added to the nuxt.config.ts as an env var